raiden/raiden_service.py

```diff
@@ -456,8 +459,15 @@ def stop(self):
 
         log.debug('Raiden Service stopped', node=pex(self.address))
 
-    def add_pending_greenlet(self, greenlet: gevent.Greenlet):
+    def add_pending_greenlet(self, greenlet: Greenlet):
+        """ Ensures an error on it crashes self/main greenlet. """
```

Comment:
Perhaps change: "Ensures an error on it" to "Ensures an error on the passed greenlet"